easybuild/easyconfigs/t/TensorFlow/TensorFlow_2.15.1_ARM_CUDA.patch

```diff
@@ -0,0 +1,56 @@
+diff -ruN a/tensorflow-2.15.1/third_party/absl/absl.eb b/tensorflow-2.15.1/third_party/absl/absl.eb
+--- a/tensorflow-2.15.1/third_party/absl/absl.eb	1970-01-01 01:00:00.000000000 +0100
++++ b/tensorflow-2.15.1/third_party/absl/absl.eb	2025-05-21 09:48:59.000000000 +0200
```

Comment:
This part seems to be accidental as this file isn't something in tensorflow but an easyconfig.

Also note that we have `absl-py-2.1.0-GCCcore-12.3.0` already, so a 2nd version might not work